src/PIL/Image.py

```diff
@@ -1279,11 +1279,14 @@ def getbands(self):
         """
         return ImageMode.getmode(self.mode).bands
 
-    def getbbox(self):
+    def getbbox(self, alpha_only=True):
```

Comment:
Make it a required keyword argument (re: https://github.com/python-pillow/Pillow/pull/6954#issuecomment-1432872156)?


```suggestion
    def getbbox(self, *, alpha_only=True):
```